vhd/packer/configure-windows-vhd.ps1

```diff
@@ -159,19 +160,6 @@ function Install-ContainerD {
     $newPath = [Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) + ";$installDir"
     [Environment]::SetEnvironmentVariable("Path", $newPath, [EnvironmentVariableTarget]::Machine)
     $env:Path += ";$installDir"
-
-    Write-Log "Registering containerd as a service"
-    & containerd.exe --register-service

```

Comment:
We still need to install containerd in the VHD in order to pull nanoserver/servercore/pause image

Reply:
or at least start it

Reply:
make sense. I was hoping we wouldn't need to duplicate the configuration set up but I guess we will need that

Reply:
You can probably get by with a `Start-Job` and not need to register this as a service.
This won't persist across reboots tho

Reply:
Also, i wonder why the Windows VHD CI job didn't trigger here...

Reply:
I've made this update. PTAL

Reply:
Looks good.
Let's make sure the containerD VHD job passes the CI job!